### PR TITLE
Improve decode_vlen_strings warning message

### PR DESCRIPTION
**Pull request by hmaarrfk**
Closes https://github.com/h5netcdf/h5netcdf/issues/132

- [ ] Closes Issue #xxxx
- [ ] Tests added
- [ ] Changes are documented in `CHANGELOG.rst`

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
LGTM! @hmaarrfk If you think, we can go without a special changelog entry, I'll merge right away. Then it will be included in the 0.13.0 release.

**Review by kmuehlbauer — APPROVED** [review]

**Comment by hmaarrfk** [comment]
I had forgotten about the changelog, but this is pretty minor so I'm happy without the entry!

**Comment by kmuehlbauer** [comment]
Thanks @hmaarrfk

**Pull request merged**
